storage_controller/src/persistence.rs

```diff
@@ -1241,6 +1242,17 @@ impl SafekeeperPersistence {
             availability_zone_id: &self.availability_zone_id,
         }
     }
+    pub(crate) fn as_describe_response(&self) -> SafekeeperDescribeResponse {
```

Comment:
Perhaps worth mentioning here that `active` is intentionally omitted.

Reply:
done